src/plugins/unified_search/public/types.ts

```diff
@@ -103,3 +103,8 @@ export interface IUnifiedSearchPluginServices extends Partial<CoreStart> {
   usageCollection?: UsageCollectionStart;
   savedObjectsManagement: SavedObjectsManagementPluginStart;
 }
+
+export enum TextBasedLanguages {
```

Comment:
I moved this enum here so we could reference it outside of Unified Search since it's referenced in public props.

Reply:
I don't think we should export this enum from unified search. Do you want to move it to esql-utils instead?

Reply:
Make sense! Moved it here: https://github.com/elastic/kibana/pull/177107/commits/981c2545ae76e73a3fbd1cffe2a7dd6fdb524b18.